lldb/docs/dil-expr-lang.ebnf

```diff
@@ -3,7 +3,9 @@
 (* This is currently a subset of the final DIL Language, matching the current
    DIL implementation. *)
 
-expression = primary_expression ;
+expression = unary_expression ;
+
+unary_expression = unary_operator primary_expression ;
```

Comment:
Need to add just plain primary_expression -- we're not forcing all expressions to have unary ops -- could still be plain variable names.